DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/compute_units/src/pipe_array.hpp

```diff
@@ -0,0 +1,44 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <CL/sycl.hpp>
+#include <utility>
+
+// Header locations and some DPC++ extensions changed between beta09 and beta10
+// Temporarily modify the code sample to accept either version
+#define BETA09 20200827
+#if __SYCL_COMPILER_VERSION <= BETA09
+  #include <CL/sycl/intel/fpga_extensions.hpp>
+  namespace INTEL = sycl::intel;  // Namespace alias for backward compatibility
+#else
+  #include <CL/sycl/INTEL/fpga_extensions.hpp>
+  using namespace sycl;
+#endif
+
+
+#include "pipe_array_internal.hpp"
+
+template <class Id, typename BaseTy, size_t depth, size_t... dims>
```

Comment:
`template <typename Id, typename BaseTy, size_t depth, size_t... dims>`

Reply:
Will be changed with other pipe_array changes.